cordova-lib/src/plugman/fetch.js

```diff
@@ -236,7 +236,7 @@ function findLocalPlugin(plugin_src, searchpath, pluginInfoProvider) {
 
     versions.forEach(function(pinfo) {
         // Ignore versions that don't satisfy the the requested version range.
-        if (!semver.satisfies(pinfo.version, versionspec)) {
+        if (!semver.satisfies(pinfo.version.replace(/-dev$/, ''), versionspec)) {
```

Comment:
This is not obvious, why this change is required. Could you please add a link to JIRA/semver docs or short explanation, why this needed.